copis/gui/panels/controller.py

```diff
@@ -35,10 +36,12 @@ class ControllerPanel(scrolled.ScrolledPanel):
         parent: Pointer to a parent wx.Frame.
     """
 
-    def __init__(self, parent, *args, **kwargs) -> None:
+    def __init__(self, parent) -> None:
         """Inits ControllerPanel with constructors."""
         super().__init__(parent, style=wx.BORDER_DEFAULT)
-        self.p = parent
+        self._parent = parent
+        self._core = self._parent.core
```

Comment:
Any reason why we're using `_parent` and `_core` with an underscore?

Reply:
I wanted to scope them to where there's used. Each class gets their own reference to `parent` and `core` so they can keep it to themselves. They're all pointers anyway.